boefjes/boefjes/plugins/kat_rdns/main.py

```diff
@@ -0,0 +1,41 @@
+import json
+import socket
+from ipaddress import ip_network
+from typing import List, Tuple, Union
+
+from boefjes.job_models import BoefjeMeta
+
+
+def run_rdns(cidr):
+    network = ip_network(cidr)
+    results = []
+
+    for ip in network.hosts():
```

Comment:
I'm having some doubts about this functionality.
If someone asks this boefje to scan a Range, two options are possible:
Scan all ip's in this range (which would duplicate the data already present if all ip's are available in the graph anyway), or do a lookup on the PTR record for that specific rage.
The first obviously could be more efficient if you want PTR info on all Ip's, than scanning all IP's one by one, but we do not have a way of limiting both of these jobs to be made.
The second is currently done (because the start IP is in the range of Ip's) but will be drowned out by all the Other less related recrods.